scripts/nlp_language_modeling/README.md

```diff
@@ -0,0 +1,40 @@
+**Support of community models**
+===============
+Take LLaMA as example.
+
+Checkpoint conversion
+---------------
+
+
+```
+python /path/to/convert_hf_llama_to_nemo.py --in-file ./ --out-file /path/to/llama-nemo-ckpt/llama-13b.nemo
+```
+Use the following commands to modify the nemo checkpoint:
+
+```
+# decompress nemo checkpoint
+tar -xvf llama-13b.nemo
+# modify model_config.yaml
+# hidden_dropout: 0.0 -> hidden_dropout: 0.1
+# compress all files
+tar -cvf model_config.yaml model_weight.ckpt tokenizer.model
+```
+Of course most of configs could be overrided when launching the training script.
+
+Reshape TP/PP
+---------------
+More configs and related explanations could be found [here](https://github.com/lhb8125/nemo-for-vivo/blob/support_llama/examples/nlp/language_modeling/megatron_change_num_partitions.py#L38).
```

Comment:
That link gives 404 and besides points to a private fork -- the official project should be used